app/scss/pages/_home.scss

```diff
@@ -15,104 +16,306 @@
+			}
+		}
+	}
+}
+
+//footer
+.footer {
+	margin-top: 75px;
+	color: #fff;
+	background-color: #14274A;
+	padding: 65px 0;
+	position: relative;
+	&::after {
+	  content: ''; 
+	  position: absolute;
+	  left: 50%;
+	  transform: translateX(-50%); 
+	  top: -95px;
+		border: 62px solid transparent; 
+		border-bottom: 38px solid #14274A;
+	}
+	&__inner {
 		display: flex;
-		flex-direction: column;
-		justify-content: center;
-		flex: 1;
-		margin-right: 70px;
+		justify-content: space-between;
 		@media (max-width : 990px) {
-			text-align: center;
-			margin-right: 0;
-			order: 2;
-			margin-top: 40px;
-		}
-		&-inner {
-			position: relative;
-			&::before {
-				content: "";
-				border: 1px solid #14274A;
-				position: absolute;
-				left: -35px;
-				height: 100%;
-				@media (max-width : 1170px) {
-					display: none;
-				}
+			flex-wrap: wrap;
+		}
+		@media (max-width : 520px) {
+			flex-direction: column;
+			align-items: center;
+		}
+	}
+	&__item {
+		@media (max-width : 520px) {
+			margin-bottom: 20px;
+		}
+		&--form {
+			@media (max-width : 990px) {
+				margin: 0 auto;
+				text-align: center;
+			}
+			@media (max-width : 672px) {
+				margin-top: 20px;
+			}
+			@media (max-width : 520px) {
+				margin-top: 0;
+			}
+		}
+		&-logo {
+			@media (max-width : 520px) {
+				display: block;
+				margin: 0 auto;
 			}
 		}
-		&-title {
-			font-family: AGaramondPro, sans-serif;
-			font-size: 50px;
-			letter-spacing: 0.02em;
-			font-weight: 700;
-			letter-spacing: 0.02em;
+		&-contacts {
+			margin-top: 20px;
+			font-size: 14px;
 			line-height: 1.1;
-			@media (max-width : 1160px) {
-				font-size: 40px;
+			@media (max-width : 520px)  {
+				text-align: center;
 			}
-			@media (max-width : 990px) {
-				font-size: 35px;
+			.contact__address {
+				margin: 0;
 			}
-			@media (max-width : 480px) {
-				line-height: 1;
+			.contact__phone {
+				display: block;
 			}
 		}
-		&-text {
-			@media (max-width : 480px) {
+	}
+	
+	&-list {
+		list-style: none;
+		font-size: 16px;
+		li {
+			margin-bottom: 10px;
+		}
+	}
+
+	.social__list {
+		list-style: none;
+		line-height: 1.7;
+		@media (max-width : 1070px) {
+			margin-right: 20px;
+		}
+		@media (max-width : 672px) {
+			margin-top: 20px;
+		}
+		@media (max-width : 520px) {
+			margin-top: 0;
+		}
+		span {
+			padding-left: 20px;
+		}
+	}
+
+	&-form {
+		margin-top: 30px;
+		display: flex;
+		@media (max-width : 990px) {
+			margin-top: 20px;
+		}
+		&__input {
+			background-color: transparent;
+			border: 2px solid #E0B973;
+			font-size: 16px;
+			padding: 0 25px;
+			border-radius: 3px;
+			height: 45px;
+			width: 245px;
+			@media (max-width : 520px) {
+				width: 200px;
+			}
+			@media (max-width : 460px) {
+				width: 180px;
+				height: 40px;
 				font-size: 12px;
 			}
+			&::placeholder {
+				color: #fff;
+			}
 		}
-		&-btn {
-			width: 170px;
-			height: 50px;
-			text-align: center;
-			@media (max-width : 990px) {
-				margin: 20px auto;
+		&__btn {
```

Comment:
Add hover

All button should have hover

Reply:
done